src/main/java/org/jabref/gui/consistency/ConsistencyCheckDialog.java

```diff
@@ -129,6 +122,35 @@ protected void updateItem(String item, boolean empty) {
                                                  setText(item);
                                              }
                                      );
+
+                    this.setOnMouseClicked(event -> {
+                        if (!isEmpty()) {
+                            TableColumn<ConsistencyMessage, String> clickedColumn = getTableColumn();
+
+                            ConsistencyMessage message = getTableRow().getItem();
+                            String cellValue = getTableColumn().getCellObservableValue(getIndex()).getValue();
+                            Optional<StandardField> optionalField = StandardField.fromName(clickedColumn.getText());
```

Comment:
This will only work for StandardFiels, but not for fields from entry types e..g biblatexApa. Better use
FieldFactory.parseField() 
that method checks all entry types for any matching field